lib/plistLib/main.js

```diff
@@ -127,7 +127,8 @@ define([
             return doc;
         },
         getType: function(value) {
-            if(value instanceof IntObject)
+            if(value instanceof IntObject
+                    || typeof value === 'number' && value === (value|0))
```

Comment:
Ah, no I just reread the spec and it is in there so I guess it's safe. One neat thing I did find (always new things to find in this spec) is that "The value produced by a && or || operator is not necessarily of type Boolean. The value produced will always be the value of one of the two operand expressions." which isn't a problem here, but not something you'd expect, either.